docs/source/manual/usermanual.rst

```diff
@@ -149,7 +149,80 @@ User management
 
 Users and organizations can be created in the on boarding flow, in the Web interface or automated. The administrator of the system can create organizations and do user management. The administrator of an organization in turn can create users within the organization. The django interface provides additional capabilities for user management via the command line, for use in an automated deployment and linkage to external user management.
 
+Adding users through a CSV file
+-------------------------------
 
+Adding multiple users at a time to OpenKAT can be done using a CSV file.
+
+How does it work?
+*****************
+
+Select the organization to which the new users will be added. On the members page click the Add member(s) menu and select Upload a CSV. This takes you to the CSV upload page.
+
+.. image:: img/csvupload.png
+  :alt: CSV upload page
+
+Download the template file, fill in the data of the users you want to add and upload them into the system. The new users will be added to the organization of your choice.
+
+.. image:: img/csvformat.png
+  :alt: CSV format
+
+How should I prepare the CSV file?
+**********************************
+
+CSV files are great when they work. Edit the downloaded template file and use a plain texteditor to make sure your CSV file contains exactly what is needed for its purpose.
+
+Each user will have its on line in the CSV file. The template has five columns: full_name, email, account_type, trusted_clearance_level, acknowledged_clearance_level.
+
+*User details:*
+
+A user is recognized by their full name and email address.
+
+* full_name : the full name of the user
+* email : a working emailadress of the user
+
+*User type:*
+
+Through the CSV upload you can add the usertypes client, admin and redteam. Read about users and roles in the `user section of the documentation <https://docs.openkat.nl/manual/usermanual.html#users>`_
+
+* account_type : client, admin or redteam
+
+*User clearance:*
+
+Clearance levels are related to the scan level of the Boefjes a user is able to dispatch. Read about this in the `scan level section of the documentation <https://docs.openkat.nl/manual/usermanual.html#scan-levels-and-indemnities>`_.
+
+The trusted_clearance_level is the level a user receives from the organization. It is the maximum level available for this user, based on the decision of the admin or superuser. The acknowledged_clearance_level is the level accepted by the user. Both can be added in the CSV file. The accepted level can be changed by the user.
+
+* trusted_clearance_level : between -1 and 4
+* accepted_clearance_level : between -1 and 4
+
+*Warnings*
+
+If the CSV file contains data that cannot be parsed OpenKAT will give a warning with the data concerned.
+
+User notification
+*****************
+
+After the CSV file has been uploaded the users receive a welcome email on their account. The link in this email allows them to create a password for their account.
+
```

Comment:
We should make sure we tell the user this only works when their OpenKAT install can actually send emails.
@Rieven has at some point written a piece of detection code that can check if the required variables are filled in in the environment. If not, we should probably tell the user adding these accounts as such.
We no outbound Email server is configured the emails are printed to std-out (I think)

Reply:
Just a thought I had, but maybe a docs page about emailing in general would be useful.

Reply:
Yes, there is a check when the user uses the link to reset password. https://github.com/minvws/nl-kat-coordination/blob/6cb366b424b3cf0dae97e0ff185394a7f66d07f3/rocky/account/views/password_reset.py#L45

Also the server must be setup to send emails, if not it will use the std-out 
<img width="518" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/30832687/abc364db-727b-4087-b3bb-2f45cce8dc6a">

Reply:
I think we should disable CSV upload when e-mail isn't configured and DEBUG isn't set with a message that e-mail needs to be configured first. The console email backend should only be used for development and testing.